aquacomputer_d5next.c

```diff
@@ -20,8 +20,8 @@
 #include <asm/unaligned.h>
 
 #define USB_VENDOR_ID_AQUACOMPUTER	0x0c70
-#define USB_PRODUCT_ID_FARBWERK 	0xf00a
-#define USB_PRODUCT_ID_QUADRO 		0xf00d
+#define USB_PRODUCT_ID_FARBWERK		0xf00a
```

Comment:
nit: please revert these two spacing changes -- it helps keep the diff small when we try to mainline this